src/components/app/App.js

```diff
@@ -1,24 +1,58 @@
 import React from 'react';
-import logo from '../../logo.svg';
-import './App.css';
+
+import { Header } from '../header/HeaderFromLecture';
+import { Footer } from '../footer/Footer';
+import TestCard, { PostCard as Card } from '../post-card/PostCard';
+import { postsList } from '../../constants'; // todo помимо константы postsList достать еще usersList
+import { usersList } from '../../constants';
+// todo: тут сделать импорт  UserCard из components/user-card/UserCard
+import {UserCard} from "../user-card/UserCard";
+import './App.scss';
+import 'bootstrap/dist/css/bootstrap.min.css';
 
 function App() {
+
+  const renderList = () => {
+    const res = [];
+
+    for (let i = 0; i < 6; i++) {
+      const item = postsList[i];
+      res.push(<Card post={item} key={item.id}/>)
+    }
+
+    return res;
+  };
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
+      <div>content will appear here soon</div>
+
+      {/*<div className="d-flex posts-container">*/}
+      {/*  {*/}
+      {/*    renderList()*/}
+      {/*  }*/}
+      {/*</div>*/}
+
+      <div className="d-flex posts-container">
+        {
+          postsList.map((item, index) => {
+            const odd = index % 2 !== 0;
+
+            console.log(odd)
+              return <Card post={item} key={item.id} hasImage={odd} />
+          })
+        }
+      </div>
+
+      <div className="d-flex posts-container">
+      {/*  todo: срендерить тут список пользователей, используя компонент UserCard */}
+          {usersList.map((user,id)=>{
+                  return  <UserCard user={user} key={id}/>
```

Comment:
I bet user object has id property, so you can use user.id as key
It's not a mistake, just keep in mind for next time. No need to fix it